build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/client_api_module.rb

```diff
@@ -450,6 +468,15 @@ def initialize
         # @return [Boolean]
         attr_accessor :deprecated
 
+        # @return [Boolean]
+        attr_accessor :endpoint_discovery_exist
```

Comment:
This isn't very descriptive...

`:endpoint_discovery_required`?
`:endpoint_discovery_available`?